doc/digital-receipt/General/receive-receipts.md

```diff
@@ -9,10 +9,42 @@ There are various ways receipts are provided and transported towards the consume
 
 ## With customer facing display/device 
 
+![receive_display-device](https://github.com/fiskaltrust/interface-doc/assets/124153755/d2c1e2eb-c966-4e56-8776-ad1f5fea17af)
```

Comment:
These links are most likely not permanent :( 
Please put the pictures into the repo and link them via a "relative link", as e.g. here: https://github.com/fiskaltrust/interface-doc/blob/master/doc/general/cash-register-integration/cash-register-integration-failure-scenarios.md?plain=1#L20